src/main/java/Duke/InvalidCommandException.java

```diff
@@ -0,0 +1,13 @@
+package Duke;
```

Comment:
Good job in keeping this class [simple](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#practice-kissing) and easy to read.